examples/comms_demo_492.py

```
@@ -0,0 +1,255 @@
+    transmission_pattern = ""
+    for i in range(20):
+        asyncio.run(sim.generate_telemetry())
+        success = sim.comms_sim.transmit_packet()
+        transmission_pattern += "." if success else "X"
+    
+    print(f"Transmission pattern: {transmission_pattern}")
+    
+    # Show Gilbert statistics
+    if sim._comms_fault:
+        fault_state = sim._comms_fault.get_fault_state()
+        print(f"\nFault State:")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)